websites/S/SimulatorFM/presence.ts

```diff
@@ -0,0 +1,62 @@
+const presence = new Presence({
+  clientId: "790721299126943744"
+}),
+browsingStamp = Math.floor(Date.now() / 1000),
+dj = document.querySelector(".m-player__presenter-name") as HTMLElement;
+
```

Comment:
this element is only get on website load, is that correct?

Reply:
Sorry for the slow reply, that element should be updated each time as it can change.

Reply:
Updated the code to reflect the expected outcome.